core/domain/exp_services_test.py

```diff
@@ -958,6 +966,11 @@ def test_update_param_changes(self):
             self.owner_id, self.EXP_ID, _get_change_list(
                 self.init_state_name, 'param_changes', self.param_changes), '')
 
+        user_exp = user_models.ExplorationUserDataModel.get_by_id(
```

Comment:
This seems like a rather odd place to put this test. Can we just limit the testing of this functionality to the new test class you created, and revert all the changes to this test class?